include/ebpf_program_attach_type_guids.h

```diff
@@ -33,7 +33,8 @@ extern "C"
 
     /** @brief The programs attached to the INETx_CONNECT hook will be invoked for
      * connect() calls on TCP or UDP sockets or when a UDP socket calls sendto() to
-     * a unique remote address/port tuple. May be scoped to a network compartment.
+     * a unique remote address/port tuple. May be scoped to a network compartment. The programs return 0 to permit the
+     * connection, and a negative integer to block them.
```

Comment:
What happens if a positive integer is returned?  Undefined behavior?  We should test it to make sure it doesn't crash or do something unexpected.

Reply:
Fixed after consulting linux samples.

Reply:
Sorry I missed this one. Programs should return 1 to indicate permit and 0 to indicate a drop. Now do we need to document what the extension would do if the program returns neither? Of course the implementation would be robust and handle that case, but do we need to document it?

Reply:
We don't need to document it.  My comment was just about making sure there is test coverage for that case.

Reply:
OK - I will add test cases for that. Resolving the comment.